### ISSUE TITLE
How to use it

### ISSUE THREAD

**Issue by xuchengchuang** [madmaxoft/VisualMassifDiff#4]
How to Use it.

I click Snapshots and import the massif.out file, and the core is generated. What kind of file should be imported? I converted the massif.out file to a vmdp file through ms_print, and I couldn't open it.

**Comment by madmaxoft** [comment]
Hello, sorry for the late reply, I was out for holiday and this program was written so long ago that I had to refresh my own memory :)

I've run into the same problem, where reading a massif.out file crashed the program. It seems Massif has changed a few things in the underlying format and an invariant my program was expecting didn't hold anymore. I fixed it as of e96b98bfb2b63c51862cbdd16bb4ebad70470d03 so it should work now. I also changed the build system from qmake to CMake, as the qmake method was no longer working for me at all. Opening the CMakeLists.txt file in QtCreator is still the easiest way to get this to run.

In short, get the latest sources, recompile and it should work for you, your initial technique was correct. Note that you may want to run valgrind's massif with `--detailed-freq=1` so that the information is as accurate as possible.
